[2016-03-25 08:18:57] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8087): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8078): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8068): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2393): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2340): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2324): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\InboxManagement\services\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 {main}  
[2016-03-25 08:18:57] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2347): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\InboxManagement\services\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2016-03-25 08:19:17] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8087): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8078): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8068): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2393): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2340): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2324): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\InboxManagement\services\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 {main}  
[2016-03-25 08:19:17] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2347): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\InboxManagement\services\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2016-03-25 08:19:18] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8087): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8078): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8068): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2393): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2340): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2324): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\InboxManagement\services\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 {main}  
[2016-03-25 08:19:18] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2347): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\InboxManagement\services\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2016-03-25 08:19:53] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8087): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8078): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8068): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2393): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2340): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2324): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\InboxManagement\services\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 {main}  
[2016-03-25 08:19:53] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2347): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\InboxManagement\services\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2016-03-25 08:20:07] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8087): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8078): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8068): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2393): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2340): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2324): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\InboxManagement\services\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 {main}  
[2016-03-25 08:20:07] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2347): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\InboxManagement\services\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2016-03-25 08:20:10] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8087): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8078): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8068): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2393): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2340): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2324): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\InboxManagement\services\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 {main}  
[2016-03-25 08:20:10] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2347): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\InboxManagement\services\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2016-03-25 08:22:11] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8087): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8078): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8068): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2393): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2340): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2324): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\InboxManagement\services\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 {main}  
[2016-03-25 08:22:11] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2347): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\InboxManagement\services\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2016-03-25 08:22:16] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#10 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#11 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#12 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#13 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8087): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#14 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8078): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#15 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(8068): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#16 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2393): Illuminate\Routing\Router->dispatch(Object(Illuminate\Http\Request))
#17 [internal function]: Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#18 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(52): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(3226): Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#20 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#21 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9797): call_user_func_array(Array, Array)
#22 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#23 C:\xampp\htdocs\InboxManagement\services\vendor\laravel\framework\src\Illuminate\Routing\Pipeline.php(32): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#24 [internal function]: Illuminate\Routing\Pipeline->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#25 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(9787): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#26 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2340): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#27 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2324): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#28 C:\xampp\htdocs\InboxManagement\services\public\index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#29 {main}  
[2016-03-25 08:22:16] production.ERROR: exception 'RuntimeException' with message 'No supported encrypter found. The cipher and / or key length are invalid.' in C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php:7631
Stack trace:
#0 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1331): Illuminate\Encryption\EncryptionServiceProvider->Illuminate\Encryption\{closure}(Object(Illuminate\Foundation\Application), Array)
#1 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build(Object(Closure), Array)
#2 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('encrypter', Array)
#3 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1388): Illuminate\Foundation\Application->make('Illuminate\\Cont...')
#4 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1365): Illuminate\Container\Container->resolveClass(Object(ReflectionParameter))
#5 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1351): Illuminate\Container\Container->getDependencies(Array, Array)
#6 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1287): Illuminate\Container\Container->build('App\\Http\\Middle...', Array)
#7 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(1845): Illuminate\Container\Container->make('App\\Http\\Middle...', Array)
#8 C:\xampp\htdocs\InboxManagement\services\bootstrap\cache\compiled.php(2347): Illuminate\Foundation\Application->make('App\\Http\\Middle...')
#9 C:\xampp\htdocs\InboxManagement\services\public\index.php(58): Illuminate\Foundation\Http\Kernel->terminate(Object(Illuminate\Http\Request), Object(Illuminate\Http\Response))
#10 {main}  
[2016-03-25 08:48:08] production.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Maximum execution time of 30 seconds exceeded' in C:\xampp\htdocs\InboxManagement\services\app\Http\Controllers\Inbox\InboxController.php:35
Stack trace:
#0 {main}  
